### PR TITLE
Add Users with unique titles

### LINKED ISSUE #2523: Add a list of users with titles (that aren't part of major groups)
@dorothy3242 said she wants to make a wiki version of the "Title List" dropdown [here](https://osu.ppy.sh/community/forums/topics/897590)

### PR DESCRIPTION
**Pull request by cl8n**
closes #2523

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/en.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+| ![][flag_KR] [W h i t e](https://osu.ppy.sh/users/685229) | Elite Mapper |
+| ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029) | osu!mania Champion |
+| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist |
+| ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | Elite Mapper |
+| ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728) | osu!mania champion |
+| ![][flag_HM] [Zallius](https://osu.ppy.sh/users/55) | Boats |
+| ![][flag_DE] [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter |
+| ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) | Elite Mapper |
+
+[flag_AT]: /wiki/shared/flag/AT.gif
```
Comment (MegaApplePi) [inline wiki/People/Users_with_unique_titles/en.md]: Did you want to add the title text for the country name here?

File: wiki/People/Users_with_unique_titles/en.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+[Featured Artists](/wiki/Featured_Artists) with osu! accounts are given the **Featured Artist** title to show their involvement.
+
+- ![][flag_US] [antiPLUR](https://osu.ppy.sh/users/7318723)
+- ![][flag_AT] [creosu](https://osu.ppy.sh/users/13958834)
+- ![][flag_US] [Dictate](https://osu.ppy.sh/users/5983379)
+- ![][flag_SE] [Helblinde](https://osu.ppy.sh/users/48053)
+- ![][flag_DE] [KIRA](https://osu.ppy.sh/users/2138310)
+- ![][flag_JP] [nakanojojo](https://osu.ppy.sh/users/10631848)
+- ![][flag_US] [onumi](https://osu.ppy.sh/users/11204867)
+- ![][flag_ES] [sotuiofficial](https://osu.ppy.sh/users/14779258)
```
Comment (Coppertine) [inline wiki/People/Users_with_unique_titles/en.md]: Refine to say this is due to "Rising Sun Trax"?

Comment (cl8n) [reply wiki/People/Users_with_unique_titles/en.md]: will put all of their fa pages here

File: wiki/People/Users_with_unique_titles/en.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
+# Users with unique titles
+
+User titles are typically associated with [user groups](/wiki/People/The_Team#user-groups), but the users listed here were given their titles for other reasons. Most of the titles were rewards for contribution or achievement, with the remaining being tokens of iconic or funny osu! history.
```
Comment (TPGPL) [inline wiki/People/Users_with_unique_titles/en.md]:
```suggestion
User titles are typically associated with [user groups](/wiki/People/The_Team#user-groups), but the users listed below were given their titles for other reasons. Most of the titles were rewards for contributions or achievements, with the remaining being tokens of iconic or funny osu! history.
```

File: wiki/People/Users_with_unique_titles/en.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+- ![][flag_KR] [Transcendence](https://osu.ppy.sh/users/903155)
+- ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029)
+- ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728)
+
+Winners of MWC 7K 2019 with the **osu!mania Champion** user title:
+
+- ![][flag_KR] [Estonians](https://osu.ppy.sh/users/7014697)
+- ![][flag_KR] [Jakads](https://osu.ppy.sh/users/259972)
+- ![][flag_KR] [NKEYBOARD](https://osu.ppy.sh/users/10132430)
+- ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029)
```
Comment (TPGPL) [inline wiki/People/Users_with_unique_titles/en.md]: I have mixed feelings about having to update the article 5 times a year. Maybe delete the installments and players and leave something like `Winners of the {TYPE} World Cup receive the **osu!{TYPE} Champion** user title` or something like this

File: wiki/People/Users_with_unique_titles/en.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+- ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057)
+- ![][flag_LY] [Soul Evans](https://osu.ppy.sh/users/4490770)
+- ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340)
+- ![][flag_KR] [W h i t e](https://osu.ppy.sh/users/685229)
+- ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661)
+- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768)
+
+### Aspire mapping contest winners
+
+The Aspire mapping contests featured a special variation of the Elite Mapper title called **Elite Mapper: Aspirant** as a prize.
```
Comment (TPGPL) [inline wiki/People/Users_with_unique_titles/en.md]:
```suggestion
The Aspire mapping contests feature a special variation of the Elite Mapper title called **Elite Mapper: Aspirant** as a prize.
```

File: wiki/People/Users_with_unique_titles/en.md (hunk 4 above)
Comment (cl8n) [reply wiki/People/Users_with_unique_titles/en.md]: this article is supposed to list all of the people, though. world cups already need other wiki things to be updated regularly

**Comment by cl8n** [comment]
forgot that one. the rest of the todos can be in future updates, this should be done for now

**Pull request merged**